src/components/MoneyRequestConfirmationList.js

```diff
@@ -434,5 +484,12 @@ export default compose(
         policyCategories: {
             key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyID}`,
         },
+        mileageRate: {
+            key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            selector: (policy) => DistanceRequestUtils.getDefaultMileageRate(policy),
```

Comment:
NAB: you can use shorthand here
```suggestion
            selector: DistanceRequestUtils.getDefaultMileageRate,
```